cluster_tools/copy_volume/copy_volume.py

```diff
@@ -195,12 +201,18 @@ def cast_type(data, dtype):
         data = vu.normalize(data)
         data *= 255
         return data.astype("uint8")
+    # check negative values for signed int
+    elif np.issubdtype(data.dtype, np.signedinteger) and not np.issubdtype(np.dtype(dtype), np.signedinteger):
```

Comment:
This is wrong. It will be triggered e.g. for the case where `input_type="int8"`,  `dtype(parameter)=uint8`, `int_to_uint=False`. Where the cast should not happen. Pass the `int_to_uint` flag to this function instead and then just use it in this condition.